src/Okta/Provider.php

```diff
@@ -44,36 +44,44 @@ protected function getOktaUrl()
         return $this->getConfig('base_url');
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getAuthServerId()
+    {
+        return $this->getConfig('auth_server_id', 'default');
```

Comment:
This should not default to `default` - per [this doc](https://support.okta.com/help/s/article/400-error-on-authentication-to-Okta-using-OIDC?language=en_US), many okta users should not have `default` for the `auth_server_id`. And you can't override this to an empty string in the config - if you send in an empty string in the config, `default` is set.

Reply:
Nevermind, looks like this was fixed in 8bdcb1ab129638daa11882168b80d3e4609d559b